lib/bad-request.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+const cowsay = require('cowsay');
+
+module.exports = function(res){
+  let defaultMessage = 'bad request';
+  res.statusCode = 400;
+  res.setHeader('Content-Type', 'text/plain');
+  res.write(cowsay.say({text: defaultMessage, f: 'bad request\ntry: localhost:3000/cowsay?text=howdy'}));
```

Comment:
While we can successfully POST and GET with valid data, there is a bug when I purposely try to get 400 errors (i.e., if I didn't send any text to the server). 

I traced it back to Line 8 of this file. If you're using the ``f`` property of the ``cowsay.say`` method, the ``f`` is so you can change the cow to a different animal and you have to write values in the ``f`` field according to the the cowsay docs, you can't put any string in there. Replace Line 8 with:

```
res.write(cowsay.say({text: defaultMessage}));
```

And your 400 errors should work fine.